python/tools/knowledge_tool.py

```diff
@@ -23,18 +23,19 @@ def execute(self, question="", **kwargs):
             else: perplexity = None
 
             # duckduckgo search
-            duckduckgo = executor.submit(duckduckgo_search.search, question)
+            # duckduckgo = executor.submit(duckduckgo_search.search, question)
```

Comment:
@nengisuls may I know why you commented the duckduckgo search. I think it was serving as a fallback to perplexity search. Same for below lines commented in this file.

Reply:
Hi I have issue 
/agent-zero/venv/lib/python3.11/site-packages/langchain_core/vectorstores/base.py:559: UserWarning: Relevance scores must be between 0 and 1, got [ (...) , -280.35173926548947)]
No relevant docs were retrieved using the relevance score threshold 0.1